flair/models/sequence_tagger_model.py

```diff
@@ -631,6 +632,7 @@ def _init_model_with_state_dict(cls, state, **kwargs):
             state,
             embeddings=state["embeddings"],
             tag_dictionary=state["tag_dictionary"],
+            tag_format=state.get("tag_format", "BIOES"),
```

Comment:
I would remove this here since "BIOES" is the default anyway when initializing a SequenceTagger.

Reply:
suggestion, to stay consistent with other state keys: add tag_format as in lines 618-624:
`tag_format = "BIOES" if "tag_format" not in state.keys() else state["tag_format"]`

and pass tag_format in super() method